tuna/go_fish.py

```diff
@@ -43,8 +43,13 @@ def parse_args():
                       nargs='?',
                       default=Library.MIOPEN,
                       type=Library,
-                      help="Specify library to run",
+                      help="Specify library to run, defaults to MIOpen",
```

Comment:
I would suggest remove the default and make it a required parameter

Reply:
cant make a positional argument required. Should i remove positional?

Reply:
The `nargs=?` above makes the positional argument optional, removing it as well as the default to make this argument required. [As described in this SO post](https://stackoverflow.com/questions/4480075/argparse-optional-positional-arguments)